adafruit_magtag/magtag.py

```diff
@@ -293,6 +300,50 @@ def set_text(self, val, index=0, auto_refresh=True):
         if auto_refresh:
             self.refresh()
 
+    def enter_deep_sleep(self, sleep_time):
+        """
+        Enter deep sleep and restart the program after a certain amount of time
```

Comment:
Please be explicit that it stops the program before deep sleeping.

Reply:
Ok, will do.

Reply:
The underlying  function `exit_and_deep_sleep_until_alarms` has `exit_and_` to make it clear it does not return. So some similar kind of name difference from the light-sleep function might be helpful to the user.

Reply:
Ok, I renamed it enter_deep_sleep_and_restart. I also changed the comment to reflect that better.